linux_os/guide/services/ssh/sshd_approved_ciphers.var

```diff
@@ -11,5 +11,10 @@ operator: equals
 interactive: false
 
 options:
+    {{#- STIG is product-dependent, so the selector has to be made available for individual products selectively -#}}
+    {{%- if product in ("rhel7", "ol7") %}}
     stig: aes128-ctr,aes192-ctr,aes256-ctr
+    {{%- elif product in ("rhel6", "ol6") %}}
+    stig: aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
+    {{%- endif %}}
```

Comment:
```suggestion
    el7stig: aes128-ctr,aes192-ctr,aes256-ctr
    el6stig: aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
```

Variables need to be user friendly. They don't need to have a build selector for individual products. Selectors bread complexity and are generally not useful for variables.

Reply:
I don't understand what you mean by "Variables ... don't need to have a build selector for individual products" while you suggest to add two build selectors for two classes of products. Could you please go into greater detail?
Regarding your approach, I generally wouldn't be against it, but in this case, it breaks backward compatibility - we use the `stig` selector for this variable in the RHEL7 STIG profile file.

Reply:
> Regarding your approach, I generally wouldn't be against it, but in this case, it breaks backward compatibility - we use the stig selector for this variable in the RHEL7 STIG profile file.

Then use the following:

```
    stig: aes128-ctr,aes192-ctr,aes256-ctr
    el6stig: aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc
```

Technically, the RHEL6 content will be considered EOL in November anyway.